src/sentry/api/endpoints/organization_searches.py

```diff
@@ -5,14 +5,15 @@
 from sentry import analytics
 from sentry.api.bases.organization import OrganizationEndpoint, OrganizationSearchPermission
 from sentry.api.serializers import serialize
-from sentry.models.savedsearch import SavedSearch
+from sentry.models.savedsearch import SavedSearch, SortOptions
 from sentry.models.search_common import SearchType
 
 
 class OrganizationSearchSerializer(serializers.Serializer):
     type = serializers.IntegerField(required=True)
     name = serializers.CharField(required=True)
     query = serializers.CharField(required=True, min_length=1)
+    sort = serializers.ChoiceField(choices=SortOptions.as_choices(), required=True)
```

Comment:
If you chose a default value you wouldn't break backwards compatibility.

Reply:
@markstory went ahead and added a default